src/Breadcrumb/index.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import styles from './Breadcrumb.scss';
+import Icon from '../Icon';
+
+const Breadcrumbs = ({
+  links, activeLabel, spacer, clickHandler,
+}) => {
+  const linkCount = links.length;
+
+  return (
+    <nav aria-label="breadcrumb">
+      <ol className={classNames(styles['list-inline'])}>
+        {links.map(({ url, label }, i) => (
+          <React.Fragment key={url}>
+            <li className={classNames(styles['list-inline-item'])}>
+              <a href={url} {...(clickHandler && { onClick: clickHandler })}>{label}</a>
+            </li>
+            {(activeLabel || ((i + 1) < linkCount)) &&
+              <li className={classNames(styles['list-inline-item'])} role="presentation" aria-label="spacer">
+                {spacer || <Icon className={['fa', 'fa-chevron-right']} />}
```

Comment:
This might be an issue with `Icon`, but I noticed that all the spacer Icons have the same ID in storybook

Reply:
It is an issue with `Icon`, but I can get around it until that issue is resolved.